packages/generator/src/sdk-metadata/generation-and-usage.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { getApiSpecificUsage, getGenerationDocumentation, getGenericUsage } from './generation-and-usage';
+import { checkUrlExists } from './links.spec';
+
+describe('sdk metadata - generation and usage content',()=>{
+  it('creates generic usage example',async()=>{
+    await expect(getGenericUsage()).resolves.toMatchSnapshot();
+  });
+
+  it('creates api specific usage for entity',async()=>{
+    await expect(getApiSpecificUsage({ npmPackageName: '@sap/dummy-package', entities : [{ className:'DummyClass' }] } as any)).resolves.toMatchSnapshot();
+  });
+
+  it('gives instruction with working link',async ()=> {
+    const url = getGenerationDocumentation().match(/(http.*)/)![1];
```

Comment:
minor: I would create a const which only returns a url and test that url to avoid using regex, which might be buggy (e.g., `http http://www.example.com`)